node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/nodeadmin/NodeAdminStateUpdater.java

```diff
@@ -56,23 +55,25 @@
     private final Clock clock;
     private final Orchestrator orchestrator;
     private final String dockerHostHostName;
+    private final Duration nodeAdminConvergeStateInterval;
 
-    private long delaysBetweenEachTickMillis = 30_000;
     private Instant lastTick;
 
     public NodeAdminStateUpdater(
-            final NodeRepository nodeRepository,
-            final NodeAdmin nodeAdmin,
+            NodeRepository nodeRepository,
+            Orchestrator orchestrator,
             StorageMaintainer storageMaintainer,
+            NodeAdmin nodeAdmin,
+            String dockerHostHostName,
             Clock clock,
-            Orchestrator orchestrator,
-            String dockerHostHostName) {
+            Duration nodeAdminConvergeStateInterval) {
         log.log(LogLevel.INFO, objectToString() + ": Creating object");
         this.nodeRepository = nodeRepository;
-        this.nodeAdmin = nodeAdmin;
-        this.clock = clock;
         this.orchestrator = orchestrator;
+        this.nodeAdmin = nodeAdmin;
         this.dockerHostHostName = dockerHostHostName;
+        this.clock = clock;
+        this.nodeAdminConvergeStateInterval = nodeAdminConvergeStateInterval;
         this.lastTick = clock.instant();
 
         specVerifierScheduler.scheduleWithFixedDelay(() ->
```

Comment:
We should wait with scheduling this until start(), could you fix that as well?

Reply:
Fixed.